.github/workflows/build-push-deploy-promrated.yaml

```diff
@@ -49,11 +49,15 @@ jobs:
         push: true
         build-args: GITHUB_SHA=${{ github.sha }}
         tags: ${{ steps.meta.outputs.tags }}
+        
+    - name: Set shortened SHA
+      id: vars
+      run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
```

Comment:
`Define docker image meta data tags` already does this. Feels like we are hacking here...

Reply:
I went with what you suggested, however the tages are in the form "name:sha" so there is still an additional step needed. Let me know which you prefer.